tests/testthat/test-mediationanalysis.R

```diff
@@ -9,7 +9,7 @@ test_that("Simple mediation analysis works", {
   options$estimator              <- "ml"
   options$errorCalculationMethod <- "standard"
   options$naAction               <- "fiml"
-  results <- jaspTools::runAnalysis("MediationAnalysis","test.csv", options)
+  results <- jaspTools::runAnalysis("MediationAnalysisInternal","test.csv", options)
```

Comment:
hm this should stay the same, i.e., callling the wrapper function, not the internal one. Could you remove these changes and make sure to rebase?